lib/ansible/modules/system/seboolean.py

```diff
@@ -49,7 +49,7 @@
     choices: [ 'yes', 'no' ]
 notes:
    - Not tested on any debian based system
-requirements: [ ]
+requirements: [ 'libselinux-python', 'libsemanage-python' ]
```

Comment:
@bcoca This matter appears to have been resolved in PR #23029, which has since been merged, and I have rebased my changes on top of this.